### PR TITLE
Request to add new bidder

### PR DESCRIPTION
**Pull request by envivoBidder**
envivoBidder is the new adapter we want to integrate with prebid.js

## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder: '<bidder name>',
  params: {
    // ...
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/
 
- PR link https://github.com/prebid/prebid.github.io/pull/1699

### CODE REVIEW

File: modules/envivoBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
+import { registerBidder } from '../src/adapters/bidderFactory';
+import { config } from '../src/config';
+import * as utils from '../src/utils';
+import {BANNER, VIDEO} from '../src/mediaTypes';
+import { ajax } from '../src/ajax';
+import {Renderer} from '../src/Renderer';
+
+const SUPPORTED_AD_TYPES = [BANNER, VIDEO];
+const BIDDER_CODE = 'envivo';
+const DOMAIN = 'http://ad.nvivo.tv/';
```
Comment (Fawke) [inline modules/envivoBidAdapter.js]: We're deprecating the use of `http` with the `prebid-3.0` release. We recommend you change this to `https` to be part of the new **prebid-3.0** release.

**Comment by Fawke** [comment]
@envivoBidder Hi, can you please update the docs on this repo? https://github.com/prebid/prebid.github.io

**Comment by Fawke** [comment]
@envivoBidder I was trying to test the video (instream) mediaType with the test params you provided in the md file. I'm getting a CORS error when I do so.
Screenshot 2019-12-11 at 7 40 57 PM

Can you make some changes at your server to resolve this? Thanks.

File: modules/envivoBidAdapter.js (hunk 1 above)
Comment (envivoBidder) [reply modules/envivoBidAdapter.js]: As you said, we have changed from http to https.

**Pull request merged**